lib/phoenix_live_view/channel.ex

```diff
@@ -48,6 +48,11 @@ defmodule Phoenix.LiveView.Channel do
     send(self(), {@prefix, :drop_upload_entries, info})
   end
 
+  def report_writer_error(pid, reason) do
+    channel_pid = self()
+    send(pid, {@prefix, :report_writer_error, channel_pid, reason})
```

Comment:
I mirrored above, is there a reason not to use `GenServer.cast`?

Reply:
I tend to have an aversion to genserver cast so honestly I didn't even think about it in these cases :)